docs/images.md

```diff
@@ -1,35 +1,73 @@
 ---
 sidebar_position: 8
```

Comment:
sidebar_position is no longer required in the file headers. The sidebar position is controlled via configuration in the sidebar.js file.

Reply:
sidebar_position removed.